src/transaction_builder.js

```diff
@@ -66,7 +66,8 @@ export class TransactionBuilder {
         this.operations    = [];
         this.signers       = [];
 
-        this.baseFee    = opts.fee || BASE_FEE;
+        this.baseFee    = (isNaN(opts.fee) ? BASE_FEE : opts.fee);
```

Comment:
The value of `opts.fee` when the variable is unset is `undefined` so I'd change `isNan` to `isUndefined`.